src/pages/search.vue

```diff
@@ -56,11 +53,25 @@ const BrowsePage = {
     const isMinScreenMd = isMinScreen('md')
     const { isVisible } = useFilterSidebarVisibility()
     const showScrollButton = inject('showScrollButton')
+    const searchStore = useSearchStore()
+
+    const query = computed(() => searchStore.query)
+    const searchType = computed(() => searchStore.searchType)
+    const searchQueryParams = computed(() => searchStore.searchQueryParams)
```

Comment:
Apparently, we do, because `searchStore.parameter` return the inner value, not the `computed`. We could also just return `searchStore` from the `setup` function, but that means that we have to use `this.searchStore.query` instead of `this.query` in options api methods, and `searchStore.query` instead of `query` in the templates. I don't see a clear advantage of that.